### ISSUE TITLE
Create VM with vsphere_guest from template and change IP address

### ISSUE THREAD

**Issue by kikicarbonell** [ansible/ansible-modules-core#3102]
##### ISSUE TYPE

Feature Idea
##### COMPONENT NAME

vsphere_guest
##### ANSIBLE VERSION

N/A
##### SUMMARY

I need a way to create virtual machine (VM) from template and change ip address in the creation process.

**Comment by reinier-pv** [comment]
:+1:

**Comment by cundy** [comment]
:+1:

**Comment by ansibot** [comment]
@chrrrles, @rhoop, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@chrrrles, @rhoop, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@chrrrles, @rhoop, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by matts-mpg** [comment]
Ended up here when googling for the exact topic. Also, changing the size of the default disk in a VM template would be super useful so once the machine is up some ansibling can be used to get the additional space partitioned and used via LVM/xfs_growfs. 

PS- This really belongs under the newer vmware_guest module.

**Comment by ansibot** [comment]
This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.

**Comment by ansibot** [comment]
This issue was migrated to https://github.com/ansible/ansible/issues/29880

**Comment by matts-mpg** [comment]
i would recommend closing this. since it was originally posted this module has been superseded by vmware_guest. also, if you are looking for either function mentioned here (provision and chang IP OR expand the disk) i ended up using the newer module and doing that via playbook. https://github.com/MindPointGroup/ansible-vmware-provisioning
